client/src/app/site/base/list-view-base.ts

```diff
@@ -68,21 +88,42 @@ export abstract class ListViewBaseComponent<V extends BaseViewModel, M extends B
         titleService: Title,
         translate: TranslateService,
         matSnackBar: MatSnackBar,
+        protected route: ActivatedRoute,
+        private storage: StorageService,
         public filterService?: BaseFilterListService<M, V>,
         public sortService?: BaseSortListService<V>
     ) {
         super(titleService, translate, matSnackBar);
         this.selectedRows = [];
+        try {
+            this.storageKey = (<Type<any>>route.component).name;
+        } catch (e) {
+            this.storageKey = '';
+        }
+    }
+
+    /**
+     * Delete the subscription if existing.
+     */
+    public ngOnDestroy(): void {
+        super.ngOnDestroy();
+        if (this.paginationSubscription) {
+            this.paginationSubscription.unsubscribe();
```

Comment:
`base-view' has a 'subscriptions' object that unssubs and clears all the subscriptions once onDestroy was reached.